README.md

```diff
@@ -10,11 +10,6 @@ Simple and lightweight base classes for web components with a beautiful API
 
 #### Installation
```

Comment:
@quarkus How should we notify people that are using element-js already? A note in the changelog/release notes should be fine?

Reply:
I would just change the readme.md and document that its now possible to install element-js without a .npmrc file ..